### PR TITLE
initial brocket deer commits. 

### PR DESCRIPTION
**Pull request by jayRyan24**
Adds the bescort methods and base-hunting data.

I put it into three separate methods because we need to know where we start within the land to know which way to go first, else we could be wandering west or east. I did this to accommodate the already implemented way things are handled. 

https://pastebin.com/eN7Jx4WT

Need testers.

### CODE REVIEW

**Comment by jayRyan24** [comment]
Just had a successful full hunt in youngs.

End of it:  https://pastebin.com/EWaY0vJ6

File: bescort.lic [hunk 1]
```diff
@@ -199,14 +199,32 @@ class Bescort
       ],
       [
         { name: 'hvaral_passport', regex: /hvaral_passport/i, description: 'Handles the gate between Hvaral and Muspari' }
+      ],
+      [
+        { name: 'brocket_1', regex: /brocket_1/i, description: 'Enter the young brocket deer hunting area.' },
```
Comment (rpherbig) [inline bescort.lic]: ❓ What do you think about `young_brocket`, etc. here instead of 1/2/3?

File: bescort.lic [hunk 2]
```diff
@@ -265,6 +283,51 @@ class Bescort
 
   private
 
+  def brocket_1(mode)
+    move 'climb fence' if mode.include?('enter')
```
Comment (rpherbig) [inline bescort.lic]: ❓ Should this do a check that you're in the right room to start?

File: bescort.lic (hunk 1 above)
Comment (jayRyan24) [reply bescort.lic]: I was trying to match the rest of the hunting areas and how we do them.

File: bescort.lic (hunk 2 above)
Comment (jayRyan24) [reply bescort.lic]: I can add that, sure.

File: bescort.lic (hunk 1 above)
Comment (jayRyan24) [reply bescort.lic]: lol, we actually have a mix of it. I changed it to young, mid, and elder.

**Comment by Sarvatt** [comment]
this looks good to me, thanks for doing his long requested hunting area

**Pull request merged**